src/java.base/share/classes/jdk/internal/util/SystemProps.java

```diff
@@ -40,6 +41,18 @@ public final class SystemProps {
     // no instances
     private SystemProps() {}
 
+    // custom java.io.tmpdir via command line
+    private static String customTmpdir;
+
+    /**
+     * check if warning for custom java.io.tmpdir is required
+     *
+     * @return a boolean value
+     */
+    public static boolean checkIfWarningRequired() {
```

Comment:
I would rename the method to `checkIoTmpdir`; it will be easier to see the purpose in the call in System.